docs/sources/static/configuration/integrations/cloudwatch-exporter-config.md

```diff
@@ -132,6 +132,16 @@ Configuration reference:
   # Optional: Disable use of FIPS endpoints. Set 'true' when running outside of USA regions.
   [fips_disabled: <boolean> | default = false]
 
+
+  # Instead of retrieving metrics on request, decouple scraping retrieves the
+  # metrics on a schedule and returns the cached metrics.
+  decoupled_scraping:
+    # Enable decoupled scraping.
+    [enabled: <boolean> | default = false ]
+
+    # How often to scrape for AWS metrics.
```

Comment:
Which is it? AWS metrics? or CloudWatch? The rest of the docs refer to it as CloudWatch metrics.

Reply:
Sorry this was written before the flow docs below. I think CloudWatch is more consistent, so I'll update this.